scripts/utils.js

```diff
@@ -34,3 +34,65 @@ export const [setLibs, getLibs] = (() => {
  * Edit above at your own risk.
  * ------------------------------------------------------------
  */
+
+/**
+ * Builds a block DOM Element from a two dimensional array
+ * @param {string} blockName name of the block
+ * @param {any} content two dimensional array or string or object of content
+ */
+function buildBlock(blockName, content) {
+  const table = Array.isArray(content) ? content : [[content]];
+  const blockEl = document.createElement('div');
+
+  blockEl.classList.add(blockName);
+  table.forEach((row) => {
+    const rowEl = document.createElement('div');
+    row.forEach((col) => {
+      const colEl = document.createElement('div');
+      const vals = col.elems || [col];
+      vals.forEach((val) => {
+        if (val) {
+          if (typeof val === 'string') {
+            colEl.innerHTML += val;
+          } else {
+            colEl.appendChild(val);
+          }
+        }
+      });
+      rowEl.appendChild(colEl);
+    });
+    blockEl.appendChild(rowEl);
+  });
+  return (blockEl);
+}
```

Comment:
Not a fan of all the nested loops here, but I am guessing this is 1:1 from biz-web?

Reply:
Yeah, it's pulled from biz web. It's going to need some looping unless we want to completely restructure the `content` param. Unless you have a major objection, I'll keep it as is